frontend/src/pages/Home/Home.tsx

```diff
@@ -14,9 +31,7 @@ class Home extends React.Component {
             <th>Image</th>
             <th>Name</th>
           </tr>
-          <Pokemon name="Carapuce" id={7} />
-          <Pokemon name="Carabaffe" id={8} />
-          <Pokemon name="Tortank" id={9} />
+          {this.state ? this.state.pokemons : ""}
```

Comment:
Ton `state` React doit toujours être défini, pour éviter de tester son existence à chaque fois. Tu peux utiliser la fonction `constructor` du lifecycle pour ça : 

```jsx
constructor(props) {
  super(props);
  this.state = {pokemons: []};
}
```

Comme ça ton state est toujours défini.

Reply:
Aussi, la bonne pratique c'est que ton state ne contienne que les données à manipuler, pas la manière de les afficher. Ca permet de séparer la _logique_ de la _présentation_.

Dans ton cas, ton objet `this.state.pokemons` pourrait contenir un objet JSON, et tu peux gérer la présentation directement la présentation comme ceci par exemple :

```jsx
render() {
  <table>
  {/* ... */}
  { this.state.pokemons.length > 0
    ? this.state.pokemons.map((pokemon) => <Pokemon {/* tes props ici*/} />)
    : null
  }
</table>
}
```